package.json

```diff
@@ -10,7 +10,8 @@
     "prebuild": "rm -rf dist",
     "prepack": "npm run build",
     "pretest": "npm run lint && npm run prettier",
-    "prettier": "prettier --check --write \"./**/**.{js,ts,md,json}\"",
+    "prettier": "prettier --check .",
```

Comment:
I noticed that `prettier` is run via a `pretest` hook so we shouldn't be using the `write` flag here